tests/test_python.py

```diff
@@ -576,3 +576,18 @@ def test_yolo_world():
         close_mosaic=1,
         trainer=WorldTrainerFromScratch,
     )
+
+
+def test_fastsam_text_prompt(image, model_name="FastSAM-s.pt", device="cpu"):
+    """Test FastSAM with FastSAMPrompt.text_prompt."""
+    model = FastSAM(model_name)
```

Comment:
Looks like you might have forgotten to add an import for `FastSAM` at the top of the file, which is why all the tests are currently failing.

```py
    def test_fastsam_text_prompt(image, model_name="FastSAM-s.pt", device="cpu"):
        """Test FastSAM with FastSAMPrompt.text_prompt."""
>       model = FastSAM(model_name)

E      NameError: name 'FastSAM' is not defined
```